ocs_ci/ocs/ui/base_ui.py

```diff
@@ -388,7 +388,12 @@ def navigate_overview_page(self):
             self.choose_expanded_mode(mode=True, locator=self.page_nav["Storage"])
         else:
             self.choose_expanded_mode(mode=True, locator=self.page_nav["Home"])
-        self.do_click(locator=self.page_nav["overview_page"])
+        if Version.coerce(self.ocp_version) <= Version.coerce("4.8"):
```

Comment:
Above this you have `if Version.coerce(self.ocp_version) >= Version.coerce("4.8"):` and it has an else which you could could be in and the second if has an else which could go under the first condition. I don't see a reason to use a new condition as you condition is the else of the other condition. Please join them like.
Need to understand the logic we want. Both conditions will be used when version is 4.8. Do you need only - is this what you wanted in first place?

Reply:
As per the conditions above, we are trying to do the below actions on UI

4.5 --> Home (Open) > Overview Page
4.6 --> Home (Open) > Overiew Page
4.7 --> Home (Open) > Overview Page
4.8 --> Home (Close) > Storage (Open) > Overview Page
4.9 --> Home (Close) > Storage (Open) > ODF tab
4.10 --> Home (Close) > Storage (Open) > ODF tab
4.11 --> Home (Close) > Storage (Open) > ODF tab

As we can see here, we are trying to click on the Overview page under Home for the versions previous to 4.8, and for version 4.8 we are trying to click on the Overview page under Storage Section, and after version 4.8 we are trying to click on Storage. Thus I think we need different conditions for other steps. But I will try to check on the UIs again and see if a better solution can be written for this.

Reply:
Ok. Still don't need 2 if statement with out relation

if < 4.8
elif = 4.8
else
That will cover all options. Just implement it in one set of if condition.

Reply:
I have made the changes as you said, could you please review it once.